scripts/travis/es-integration-test.sh

```diff
@@ -9,3 +9,14 @@ STORAGE=elasticsearch make storage-integration-test
 make index-cleaner-integration-test
 
 docker kill $CID
+
+make build-ui
```

Comment:
why does this need the UI? It should only need query service API

Reply:
I received errors when I tried to compile the query service alone,.

Reply:
we build query service for crossdock using a mock UI, check that

Reply:
I'll check it, Thanks!